packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -660,13 +618,34 @@ export class AngularCompilerPlugin {
         }
       }
 
+      let ngccProcessor: NgccProcessor | undefined;
+      let ngcc: typeof import ('@angular/compiler-cli/ngcc') | undefined;
+      try {
+        // this is done for the sole reason that @ngtools/webpack
+        // support versions of Angular that don't have NGCC API
+        if (this._compilerOptions.enableIvy) {
+          ngcc = require('@angular/compiler-cli/ngcc');
+        }
+      } catch {
+      }
+
+      if (ngcc) {
+        const mainFields = getMainFields(this._compilerOptions, this._platform, false);
```

Comment:
This whole block can be moved into the `if (this._compilerOptions.enableIvy)` block above and eliminate the extra conditional as well as one less local variable in the function's scope.